proof/refine/ARM_HYP/CNodeInv_R.thy

```diff
@@ -8590,6 +8601,14 @@ lemma cteMove_urz [wp]:
   apply auto
   done
 
+crunches updateMDB
+  for valid_bitmaps[wp]: valid_bitmaps
+  (rule: valid_bitmaps_lift)
+
+lemma haskell_assert_inv:
+  "haskell_assert Q L \<lbrace>P\<rbrace>"
+  by wpsimp
```

Comment:
This is another candidate for arch split cleanup. I was about to suggest to move it to Lib, but I think that will break a whole lot of stuff, because some places rely on it in weird ways. Maybe add a `FIXME: arch_split` here so that we see it then.